README.md

```diff
@@ -1,6 +1,11 @@
 # Reference Implementation for Mastercard Transaction APIs for Acquirers
 
 [![](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Mastercard/transaction-api-reference/blob/master/LICENSE)
```

Comment:
Hyperlink for `LICENSE` isn't working. I think branch name should be `main` instead of `master`.